dimager/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class ImagerProfileConfig(AppConfig):
+    name = 'imager_profile'
+    verbose_name = 'Imager Profile'
+
+    def ready(self):
+        from handlers import create_profile
```

Comment:
Probably better simply to import `handlers` rather than importing a single symbol from that module.  That way, as you add new handlers, they will automatically be picked up.